src/python/pants/backend/python/target_types.py

```diff
@@ -573,7 +573,7 @@ class PexEntryPointsField(StringSequenceField, AsyncFieldMixin):
         "The entry points for each binary, i.e. what gets run when when executing `./my_app.pex.`"
         "\n\n"
         "Use a file name, relative to the BUILD file, like `app.py`. You can also set the "
-        "function to run, like `app.py:func`.\n"
+        "function to run, like `app.py:func`.\n\n"
```

Comment:
Ah, this is fine, but probably better to have this sentence and the next one be part of the same paragraph. They're highly related.